core/src/main/boilerplate/pureconfig/ProductReaders.scala.template

```diff
@@ -26,8 +34,16 @@ trait ProductReaders {
         a1Result
       }
   }
-
-  [2..22#final def forProduct1[B, [#A0#]]([#keyA0: String#])(f: ([#A0#]) => B)(implicit
+  [2..22#
+  /**
+   * Builds a `ConfigReader` for an object created from the values of 1 keys.
+   *
+   * @param f the function converting the tuple of values to the target object
```

Comment:
Nitpick: I may be wrong, but I don't think it's a tuple so much as an argument list. Were it a tuple, it'd have to be written as `f: ((A0, A1)) => B`

I suggest wording it as `@param f the function constructing the target object from the read config values`

Reply:
Indeed, they're not tuples. Corrected it in 766a323.